### DISCUSSION TITLE
How to Enable Github Sponsor

### DISCUSSION THREAD

**Discussion by UjjwalSainiCDC** [community/community#135080] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I want to know how i enable my sponsor button

**Comment by UjjwalSaini07** [comment, marked as answer]
Hii there,
You would find the "Your Sponsors" option once you click on your Github profile icon, then you would be redirected to stripe in further process where you need to fill your bank account details and once all your details are verified then you are good to go and start ur journey as developer and also make great project so that People make sponsor u i also provide screnshot for the same for your help take help from that as well.

Image ref : 

![image](https://github.com/user-attachments/assets/896ea7ed-4572-409a-958f-a1b75d9c8322)

![image](https://github.com/user-attachments/assets/aed7776e-075c-4f9c-81df-7388a1ad6711)

![image](https://github.com/user-attachments/assets/bc6dd521-b2fd-44e7-9594-86bde8cfcf92)

**Comment by UjjwalSainiCDC** [comment]
ohh Great i find it Thanx a lot @UjjwalSaini07 i fill details and switch on the sponsor button
